src/bootstrap/src/core/config/config.rs

```diff
@@ -1575,6 +1632,24 @@ impl Config {
             toml.profile = Some("dist".into());
         }
 
+        // Reverse the list to ensure the last added config extension remains the most dominant.
+        // For example, given ["a.toml", "b.toml"], "b.toml" should take precedence over "a.toml".
+        //
+        // This must be handled before applying the `profile` since `include`s should always take
+        // precedence over `profile`s.
+        for include_path in toml.include.clone().unwrap_or_default().iter().rev() {
+            let included_toml = get_toml(include_path).unwrap_or_else(|e| {
+                eprintln!("ERROR: Failed to parse '{}': {e}", include_path.display());
+                exit!(2);
+            });
+            toml.merge(
+                Some(config.src.join(include_path)),
```

Comment:
This should be resolved relative to the top-level config path. Otherwise it would be inconsistent with how we resolve relative paths in the nested configs. Plus the top-level config might not be in the src directory, you can pass an arbitrary path to the config.

Reply:
Oops, I thought `config.src` was the top-level path of `config.toml` for a second, sorry!